example/src/containers/ComponentsPage.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import * as Redux from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as actions from '../actions/ComponentActions';
+import { IDocumentAppState, IDocument } from '../Types';
+
+import Heading from '../components/Heading';
+import Subheading from '../components/Subheading';
+import Properties from '../components/Properties';
+import CodeExample from '../components/CodeExample';
+
+import '../styles/components-page.scss';
+import * as styles from '../styles/global-styles.scss';
+
+export interface IProps {
+  state: IDocument[];
+  actions: typeof actions.requestComponents;
+  routeParams: any;
+}
+
+class ComponentsPage extends React.Component<IProps, any> {
+  render() {
+    const currentComponent = this.props.routeParams.component;
+    const currentState = this.props.state.find((c) => { return c.id === currentComponent; });
```

Comment:
avoid using single letter parameters whenever possible. Just makes it a little easier for someone that is unfamiliar with the app to see what's going on.